tools/dependencies/checkLicense.sh

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 
-./mvnw dependency:copy-dependencies -DincludeScope=runtime -DoutputDirectory=/tmp/seatunnel-dependencies
```

Comment:
hi, we don't  need modify this, we only consider the runtime scope

Reply:
Sorry, this is the test code.